taxonomies/categories.txt

```diff
@@ -40632,6 +40652,7 @@ ciqual_food_name:en:Muesli, crunchy, with chocolate, with or without fruits, for
 ciqual_food_name:fr:Muesli croustillant au chocolat, avec ou sans fruits, enrichi en vitamines et minéraux
 
 <en:Mueslis
+<en:Cereales with fruits
```

Comment:
```suggestion
<en:Cereals with fruits
```

Reply:
fixed manually since the suggestion was not applicable from the web